sample/pubspec.lock

```diff
@@ -21,7 +21,7 @@ packages:
       path: ".."
       relative: true
     source: path
-    version: "0.0.20"
+    version: "0.0.11"
```

Comment:
You should set the version to 0.0.12 and update the CHANGELOG file so it can be correctly published in Pub.dev.